src/rpc/net.cpp

```diff
@@ -122,15 +122,15 @@ static RPCHelpMan getpeerinfo()
                     {RPCResult::Type::NUM, "bytessent", "The total bytes sent"},
                     {RPCResult::Type::NUM, "bytesrecv", "The total bytes received"},
                     {RPCResult::Type::NUM_TIME, "conntime", "The " + UNIX_EPOCH_TIME + " of the connection"},
-                    {RPCResult::Type::NUM, "timeoffset", "The time offset in seconds"},
+                    {RPCResult::Type::NUM, "timeoffset", /*optional=*/true, "The time offset in seconds"},
```

Comment:
Not sure if changing the type to optional is a refactor. There are already several conflicting pulls out there trying to change the types here from optional back or forth, so I think it would be better to not have another one. For now it might be best to make this a "real" refactor, keep the default value, and let other pulls decide if it is wrong or not.

Reply:
Rebased this on the now merged #26515, so the changing to optional was no longer needed.